consensus/reactor_test.go

```diff
@@ -669,7 +667,7 @@ func timeoutWaitGroup(t *testing.T, n int, f func(int), css []*State) {
 
 	// we're running many nodes in-process, possibly in in a virtual machine,
 	// and spewing debug messages - making a block could take a while,
-	timeout := time.Second * 300
+	timeout := time.Second * 60
```

Comment:
what's the reason for changing this?

Reply:
I thought we've increased it because the tests were failing, no?

Reply:
sorry yes I reverted it